docs/source/userguide/command_line.rst

```diff
@@ -1,508 +1,520 @@
-interfaces. This command has two modes; ``implemented`` and ``general``. Implemented
-mode searches for plugins of each interface which have been created throughout GeoIPS
-packages, or a certain package. General mode retrieves a listing of native GeoIPS
-Interfaces, which users can then create their own plugins using those interfaces.
-General Mode cannot be package specific.
-Information included when calling this command in implemented mode is:
+For example:
 
-    * GeoIPS Package
-    * Interface Type
-    * Interface Name
+.. code-block:: bash
 
-Information included when calling this command in general mode is:
+    geoips list algorithms
 
-    * GeoIPS Package
-    * Interface Type
-    * Interface Name
-    * Supported Families
-    * Docstring
-    * Absolute Path
+You can also filter by package name with ``--package_name`` or ``-p``. For example:
 
-For an example of how to run both modes of this command, see below.
+.. code-block:: bash
 
-Implemented Mode Options
-::
+    geoips list interfaces --package_name geoips
+    geoips list <interface_name> -p <package_name>
 
-    geoips list interfaces -i
-    geoips list interfaces -i -p <package_name>
-    geoips list interfaces -p <package_name>
+You can use aliases to run this command.
 
-General Mode
-::
+For example:
 
-    geoips list interfaces
+.. code-block:: bash
 
-.. _geoips_list_packages:
+    geoips ls alg
+    geoips ls algs
+    geoips list algorithm
+    geoips list algorithms
 
-:ref:`geoips list packages <geoips_list_packages>`
+``list packages`` is a sub-command of list.
 
-``list packages`` (or ``list pkgs``) is a list sub-command which retrieves a listing of
-GeoIPS Packages, alongside the information shown below.
+It returns a list of GeoIPS Packages with the following package information:
 
-    * GeoIPS Package
-    * Docstring
-    * Package Path
-    * Version Number
+* Package Name
+* Docstring
+* Package Path
+* Version Number
 
 For an example of how to run this command, see below. Notice the use of aliases in case
 you want to use these commands in shorthand style.
-::
 
-    geoips ls pkgs
+.. code-block:: bash
+
     geoips list packages
 
-.. _geoips_list_plugins:
+This command uses aliases. For example:
 
-:ref:`geoips list plugins <geoips_list_plugins>`
+.. code-block:: bash
 
-``list plugins`` (or ``list plgs``) is a list sub-command which retrieves a listing of
-plugins found within all, or a certain GeoIPS package. Information included when calling
-this command is:
+    geoips ls pkgs
 
-    * GeoIPS Package
-    * Interface Name
-    * Interface Type
-    * Family
-    * Plugin Name
-    * Source Names
-    * Relative Path
+list plugins
+^^^^^^^^^^^^
 
-For an example of how to run this command, see below. Notice the use of aliases in case
-you want to use these commands in shorthand style. One of the commands below lists
-plugins from a certain GeoIPS package.
-::
+``list plugins`` is a sub-command of list.
```

Comment:
This is stated in quite a few places. I don't think we need to state "Y is a subcommand of X". That should be obvious from the documentation flow I think.

Reply:
Fair point.